tests/format/misc/errors/js/explicit-resource-management/invalid-using-binding-let.js

```diff
@@ -1,3 +0,0 @@
-{
-  using let = h();
-}
```

Comment:
Why are we removing this test?

Reply:
If it's not throwing, we can remove `typescript` parser for this test?

Reply:
We should also ask the parser to throw.

Reply:
https://github.com/typescript-eslint/typescript-eslint/issues/7554

Reply:
thanks